js/site-links.js

```diff
@@ -0,0 +1,17 @@
+import $ from 'jquery';
+
+$(() => {
+  const $siteLinks = $('.id7-site-links');
+  const $focusableElements = $(':focusable');
```

Comment:
I'd rather we calculated this array (and the other consts I guess) only when the Esc key was pressed, rather than on every page load, because it's not a cheap lookup (iterating through _all_ elements in the DOM and filtering in JS) so may as well only do it for the fraction of a % who need this

Reply:
Complied